system/modules/HTMLHelper/models/HTMLHelper.cfc

```diff
@@ -1670,8 +1669,10 @@ component extends="coldbox.system.FrameworkSupertype" accessors=true singleton{
 				if( len( arguments.nameColumn ) ){
 					if( structKeyExists( val[ x ], arguments.nameColumn ) ){ thisName = val[ x ][nameColumn]; }
 				}
-				else{
-					if( structKeyExists( val[ x ], arguments.column ) ){ thisName = val[ x ][column]; }
+				
+				// If thisName is still the default, use the content of thisValue as the name
```

Comment:
What about this statement:
`if( structKeyExists( val[ x ], arguments.column ) ){ thisName = val[ x ][column]; }`

This checks if the column exists in the struct, if it does is uses it as the default value.

Reply:
Read the other pull request.
This is not what we want.

This would override other possible names/values we should use instead.
This is a BUG, this else is a bug